D2547.md

```diff
@@ -1853,11 +1844,39 @@ This more explicit syntax can be used to apply multiple constraints to the retur
 ```cpp
 template<typename R, typename T>
 requires decay_copyable<R> && some_concept<std::decay_t<R>>
-virtual R some_getter(const T& x) = 0;
+R some_getter(const T& x) customisable;
 ```
 
 # Additional Design Discussions
 
+## Syntax
+
+This version of the proposal introduces the `customisable` contextual keyword,
```

Comment:
> This version of the proposal _uses_ the `customisable` ...

That way we don't have to change this word when we get to R2 (since it was R1 that "introduced" the `customisable` keyword).